README.md

```diff
@@ -26,6 +26,10 @@ QCon](http://www.youtube.com/watch?v=wlvKAT7SZIQ) (voted best talk).
 ### Example
 
 ```r
+if (!require("RcppSimdJson")) install.packages("RcppSimdJson")
```

Comment:
On balance, no.

In a README.md conciseness trumps completeness.  It is also the top-level README _for the package_.  We will assume people can install it.